coq/README.md

```diff
@@ -0,0 +1,24 @@
+## Install
+
+- Install Coq using OPAM: `opam install coq` (More detail information is [here](https://coq.inria.fr/opam-using.html))
+- Use Emacs, not CoqIDE:
```

Comment:
One last thing.  I'd like not to discourage coqide here.  Please say you have two options, and just give a link to coqide, and explain emacs in details.

Besides, LGTM